fre/make/create_docker_script.py

```diff
@@ -1,22 +1,34 @@
 '''
-TODO- make docstring
+Create a Dockerfile and container build script, and generate a model container.
 '''
 
 import os
-import sys
 import logging
-fre_logger = logging.getLogger(__name__)
 
 import subprocess
-from pathlib import Path
 
-from .gfdlfremake import varsfre, targetfre, yamlfre, buildDocker
 import fre.yamltools.combine_yamls_script as cy
+from .gfdlfremake import varsfre, targetfre, yamlfre, buildDocker
+
+fre_logger = logging.getLogger(__name__)
 
 def dockerfile_create(yamlfile, platform, target, execute, skip_format_transfer):
-    srcDir="src"
-    checkoutScriptName = "checkout.sh"
-    baremetalRun = False # This is needed if there are no bare metal runs
+    """Create the dockerfile and container build script for a container build
+
+    :param yamlfile: Model compile YAML file
+    :type yamlfile: str
+    :param platform: FRE platform
+    :type platform: str
+    :param target: Predefined FRE targets
+    :type target: str
+    :param execute: Use this to run the created checkout script
+    :type execute: bool
+    :param skip_format_transfer: Skip the container format conversion to a .sif file.
```

Comment:
Maybe it should be made clear at the top of the docstring that the image will be converted to a Singular .sif image